schema.sql

```diff
@@ -31,10 +31,34 @@ name varchar(100),
 PRIMARY KEY (id)
 );
 
-CREATE TABLE VETS (
+CREATE TABLE vets (
 id SERIAL,
 name varchar(100),
 age INT,
 date_of_graduation DATE,
 PRIMARY KEY (id)
+);
+
+CREATE TABLE specialties (
```

Comment:
- [ ] According to the requirements, the name of this table should be ```specializations``` let's try to use the same name here.